anchors/anchor.go

```diff
@@ -34,7 +36,7 @@ type AnchorID [AnchorIDLength]byte
 type Config interface {
 	GetEthereumDefaultAccountName() string
 	GetEthereumContextWaitTimeout() time.Duration
-	GetContractAddress(address string) common.Address
+	GetContractAddress(contractName config.ContractName) common.Address
```

Comment:
do we need a new type for this?

Reply:
I kept running into problems where the contracts weren't initialized properly, this prevents that.